Obsidian.API/_Types/ChatMessage.cs

```diff
@@ -41,11 +44,91 @@ public IEnumerable<ChatMessage> GetExtras()
         }
     }
 
+    public static implicit operator ChatMessage(string text) => Simple(text);
+
+    /// <summary>
+    /// Adds the the right <see cref="ChatMessage"/> to the <see cref="Extra"/> of the left <see cref="ChatMessage"/>.
+    /// </summary>
+    /// <param name="a">The left chat message on which the right one gets appended</param>
+    /// <param name="b">The right chat message which will be appended</param>
+    /// <returns>The current chat message. Equally to the left chat message</returns>
+    public static ChatMessage operator +(ChatMessage a, ChatMessage b) => a.AddExtra(b);
+
+    /// <summary>
+    /// Adds the given <see cref="ChatColor"/> to the text of the given <see cref="ChatMessage"/>.
+    /// </summary>
+    /// <param name="a">The message on which the chat color gets appended</param>
+    /// <param name="b">The chat color which will be appended</param>
+    /// <returns>The current chat message. Equally to the given chat message</returns>
+    public static ChatMessage operator +(ChatMessage a, ChatColor b) => a.AppendColor(b);
+
+    /// <summary>
+    /// Creates a new <see cref="ChatMessage"/> object with plain text.
+    /// </summary>
+    /// <param name="text">The text of the <see cref="ChatMessage"/></param>
+    /// <param name="reformat">Whether to reformat the text via <see cref="ReformatFormattingPrefixes"/> or not</param>
+    /// <returns>The created <see cref="ChatMessage"/> object</returns>
+    public static ChatMessage Simple(string text, bool reformat = true) => new() { Text = reformat ? ReformatFormattingPrefixes(text) : text };
```

Comment:
I'm not completely sold on this idea. I would like some input from @Naamloos / @Tides here as well. What I don't like about this is how it's going to be used. Look at the following examples:
```cs
var msg = ChatMessage.Simple("&1 Hello World! §r"); // We will just have to remember that both & and § are supported by default, cool.
```
```cs
var msg = ChatMessage.Simple("&1 Hello World! §r", true); // What does the `true` do here? This is going to be confusing.
```
```cs
var msg = ChatMessage.Simple("&1 Hello World! §r", reformat: true); // I guess this helps a little, but at this point I'm thinking that we should just have a separate method for this.
```

Reply:
I agree on the separate method. ChatMessage.Simple for simple chat messages, and perhaps ChatMessage.Parsed or ChatMessage.ColorFormatted for the `&` notation? (could use better naming)

Reply:
Well, both are 'ColorFormatted', but one uses 'Bukkit Color Codes', so that should be reflected in the name I suppose.

Reply:
@Naamloos , how about:
```cs
public static ChatMessage Simple(string text) => new() { Text = text };
public static ChatMessage SimpleBukkit(string text) => new() { Text = ReformatAmpersandPrefixes(text) };
public static ChatMessage Simple(string text, ChatColor color) => new() { Text = $"{color}{text}" };
public static ChatMessage SimpleBukkit(string text, ChatColor color) => new() { Text = $"{color}{ReformatAmpersandPrefixes(text)}" };
```

Reply:
perhaps, though you do know you can reuse methods right? :p looks good

Reply:
Update: we have an ongoing discussion on the naming still (in the public discord server). This is a tough one 😓